spec/services/populi_service_spec.rb

```diff
@@ -130,11 +134,13 @@
         end
 
         it 'provides error message with wrong course_meeting_id' do
-          course_offering_id = "10547884"
-          enrollment_id = "76297621"
-          status = "PRESENT"
-          course_meeting_id = "531"
-          response = @populi.update_student_attendance(course_offering_id, enrollment_id, course_meeting_id, status)
+          current_academic_term = @populi.current_academic_term
+          updated_course_offering = @populi.courseofferings_by_term(current_academic_term[:id])[:data].first[:id]
+          enrollment_id = @populi.enrollments(updated_course_offering)[:data].first[:id]
+          status = "present"
+          course_meeting_id = "531" # always a four digit code in the API
```

Comment:
4 digits or 3 digits?

Reply:
In this situation we intentionally incorrectly stated the course_meeting_id as a 3 digit number since we want this test to show the error related to the course_meeting not being found.